l3backend/l3backend-color.dtx

```diff
@@ -1264,6 +1264,85 @@
 %</package>
 %    \end{macrocode}
 %
+% \subsection{Font handling integration}
+%
+% In \LuaTeX{} these colors should also be usable to color fonts, so
+% \texttt{luaotfload} color handling is extended to include these.
+%
+%    \begin{macrocode}
+%<*luatex>
```

Comment:
Do we need two sets of guards? I'd imagine we can just use `lua`.

Reply:
The idea was that these are specific to the `luatex` backend and therefore semantically belong there. (Also staying consistent with `header` and `l3backend-dvips.pro`), but just using `lua` should work too.

Reply:
I think in other places (e.g. `l3luatex`) we only have one guard.

Reply:
Ok, I removed the `luatex` guard.